helm/flink-kubernetes-operator/templates/flink-operator.yaml

```diff
@@ -79,7 +79,13 @@ spec:
           {{- end }}
           env:
             - name: OPERATOR_NAMESPACE
-              value: {{ .Release.Namespace }}
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: WATCH_NAMESPACES
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.annotations['olm.targetNamespaces']
```

Comment:
We should try to generalise this somehow. Having `WATCH_NAMESPACES` as overrides are ok, ingesting them from the 'olm.targetNamespaces' is not.

Reply:
I can remove this and work out how to inject it into the csv if you prefer ?

Reply:
I don't mind if there's no Helm support for this, if you can injest it using olm.

Reply:
Can we remove this part of the logic? 

```
- name: WATCH_NAMESPACES
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.annotations['olm.targetNamespaces']
```

Reply:
done